integration_testing/features/super-admin/super-admin-import-individual-users.feature

```diff
@@ -0,0 +1,152 @@
+Feature: Import individual users
+
+	Background:
+		Given Kolibri is not installed on my device
+			And the Kolibri installer is downloaded to my device
+			And Kolibri version 0.15 is installed on another device in my network
+			And I have a local wi-fi connection
+		When I install Kolibri
+			And I open Kolibri in my browser
+		Then I see *Please select the default language for Kolibri*
+		When I click *Continue*
+		Then I see *Getting started*
+		When I select *Advanced Setup*
+			And I click *Continue*
+		Then I see *Device name*
+			And I see <My device's name>
+		When I click *Continue*
+		Then I see *Initialize device*
```

Comment:
We changed the wording here, check the latest on the Figma prototype:
https://www.figma.com/file/w3raYcGRUhhaVm1Of8Dcvo?node-id=229:63#94924868

Reply:
Ok, I've changed 'Initialize device' to 'Select a facility setup for this device' here and in subsequent lines.

Reply:
Good, push both changes in a new commit and I'll approve! 👍🏽